include/rdma/fi_eq.h

```diff
@@ -76,8 +78,13 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
+#ifndef _WIN32
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
+#else /* _WIN32 */
+	CRITICAL_SECTION	*mutex;
+	CONDITION_VARIABLE	*cond;
+#endif /* _WIN32 */
```

Comment:
I believe critical sections and mutexes are different on windows.  Rather than fitting critical sections under this structure, I wonder if a better approach isn't to create a new critical-section wait object that's windows specific to support critical sections, and leave this structure for mutexes.  For windows support, the entire structure can be commented out.

Please note that we need to update the man pages for windows support, to document new structures and indicate which structures or options are limited to a specific OS.  I will bring this topic up at the next ofiwg as to how we want to deal with documenting OS specific structures/features in general.